src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java

```diff
@@ -143,6 +143,11 @@
      */
     public final boolean removeIrrelevantGroups;
 
+    /**
+     *  Cache of the Active Directory plugin
+     */
+    public CacheConfiguration cache;
```

Comment:
I think this should be initialized to a valid value (perhaps NO_CACHE) or something - but then the code does not have to check `if (cache != null) { /* do something */ }`